process/logsevents/interface.go

```diff
@@ -19,11 +18,13 @@ type argsProcessEvent struct {
 }
 
 type argOutputProcessEvent struct {
-	identifier string
-	value      string
-	tokenInfo  *data.TokenInfo
-	delegator  *data.Delegator
-	processed  bool
+	identifier      string
+	value           string
+	receiver        string
+	receiverShardID uint32
```

Comment:
place `receiverShardID` before `processed` -> mem alignment

Reply:
this will be refactored in the next PR